src/ol/source/imagevectorsource.js

```diff
@@ -58,6 +58,12 @@ ol.source.ImageVector = function(options) {
 
   /**
    * @private
+   * @type {number}
+   */
+  this.renderBuffer_ = options.renderBuffer || 100;
```

Comment:
Thanks @fredj - a `renderBuffer` of 0 definitely makes sense for polygon rendering. Updated.